arches_component_lab/src/arches_component_lab/cards/api.ts

```diff
@@ -40,34 +42,66 @@ export const fetchTileData = async (
     }
 };
 
-export const upsertTile = async (
+export const fetchCardXNodeXWidgetDataFromNodeGroup = async (
     graphSlug: string,
     nodegroupAlias: string,
-    data: Record<string, unknown>,
-    tileId?: string,
 ) => {
-    const args = [graphSlug, nodegroupAlias];
+    const response = await fetch(
```

Comment:
it still doesn't look like default node config values are being used in the absence of card_x_node_x_widget config values.  I'm not sure where the appropriate location to resolve this is, but but as high upstream as possible would be nice.